interface/usergroup/user_info.php

```diff
@@ -29,15 +29,15 @@ function update_password()
             iziToast.warning({
                 title: 'Warning',
                 icon: "fa fa-warning",
-                message: 'Password field cannot be empty'
+                message: 'Pass Phrase field cannot be empty'
             });
         }else {
 
             if(curPass === newPass){
                 iziToast.warning({
                     title: 'Warning',
                     icon: "fa fa-warning",
-                    message: 'Old password cannot be same with new one'
+                    message: 'Old Pass Phrase cannot be same with new one'
                 });
             }else {
                 console.log("asdasd");
```

Comment:
Oh,  I did catch this one ... this is a junk message.  Please fix to something that is meaningful.

Reply:
Hahahaha i did that during debugging alsways fun of that. Removing that right away